subworkflows/local/run_read_coverage/main.nf

```diff
@@ -48,7 +48,7 @@ workflow RUN_READ_COVERAGE {
         ch_align_bam    = SE_MAPPING.out.mapped_bam
 
     }
-    else if ( platform == "illumina" ) {
+    else if ( params.reads_type == "illumina" ) {
```

Comment:
```suggestion
    else if ( params.reads_type in ["illumina"] ) {
```

Hypothetically would you want to catch e.g. MBI/DNBSeq here?

Reply:
Hypothetically, yes. We expecting any of that to come through?

Reply:
Not to my knowledge - thinking about external users of the pipeline, really!